<?xml version="1.0" encoding="UTF-8"?>
<launch>
<param name="robot_description" command="$(find xacro)/xacro $(find projet2022)/urdf/turtlebot3.urdf.xacro " />
<!-- Run one mybot_teleop node -->
<node pkg="projet2022" name="teleop2" type="mybot_teleop.py" 
        output="screen" required="true"/>
<!--Run one follow node -->
<!--node pkg="projet2022" name="follow" type="challenge1.py" 
        output="screen" required="true"/-->
<!--node pkg="projet2022" name="corridor" type="challenge2.py" 
        output="screen" required="true"/-->
<node pkg="projet2022" name="follow" type="challenge3.py" 
        output="screen" required="true"/>

<!-- RVIZ -->
<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/>
<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
<node name="rviz" pkg="rviz" type="rviz" args="-d $(find projet2022)/rviz/config.rviz"/>

<!-- Parametre -->
<param name = "linear_scale" type = "double" value = "0.1" />
<param name = "angular_scale" type = "double" value = "0.35" />
<param name = "topic_cmd_vel" type = "string" value = "/cmd_vel"/>
<param name = "threshold" type = "double" value = "0.3"/>

</launch>
